src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/models/assessRisksAndNeeds/ArnRiskSummary.kt

```diff
@@ -12,19 +12,20 @@ data class ArnRiskSummary(
   val riskInCommunity: Map<String, List<String>>? = null,
   val riskInCustody: Map<String, List<String>>? = null,
 ) {
-  fun toRiskSummary(): RiskSummary = RiskSummary(
-    whoIsAtRisk = this.whoIsAtRisk,
-    natureOfRisk = this.natureOfRisk,
-    riskImminence = this.riskImminence,
-    riskIncreaseFactors = this.riskIncreaseFactors,
-    riskMitigationFactors = this.riskMitigationFactors,
-    overallRiskLevel = this.overallRiskLevel,
-    riskInCommunity = if (!this.riskInCommunity.isNullOrEmpty()) toRiskInContext(this.riskInCommunity) else null,
-    riskInCustody = if (!this.riskInCustody.isNullOrEmpty()) toRiskInContext(this.riskInCustody) else null,
-  )
+  fun toRiskSummary(): RiskSummary =
+    RiskSummary(
+      whoIsAtRisk = this.whoIsAtRisk,
+      natureOfRisk = this.natureOfRisk,
+      riskImminence = this.riskImminence,
+      riskIncreaseFactors = this.riskIncreaseFactors,
+      riskMitigationFactors = this.riskMitigationFactors,
+      overallRiskLevel = this.overallRiskLevel,
+      riskInCommunity = if (!this.riskInCommunity.isNullOrEmpty()) toRiskInContext(this.riskInCommunity) else null,
+      riskInCustody = if (!this.riskInCustody.isNullOrEmpty()) toRiskInContext(this.riskInCustody) else null,
+    )
 
   private fun toRiskInContext(arnRiskInContext: Map<String, List<String>>): Map<String, String> {
-    var riskInContext = mutableMapOf<String, String>()
+    val riskInContext = mutableMapOf<String, String>()
```

Comment:
If it's a mutable map, should this not be var and not val?

Reply:
val prevent reassign value to the variable. It does not prevent mutableMap to change it's content.

Reply:
Great answer, thanks